src/js_object_accessor.hpp

```diff
@@ -235,7 +242,7 @@ class NativeAccessor {
     }
 
     bool is_same_dictionary(realm::object_store::Dictionary const& dictionary, ValueType const& value) const {
-        throw std::runtime_error("'Dictionary' type support is not implemented yet");
+        return false;
```

Comment:
Is there no way we can compare to dictionaries?

Reply:
We can but we don't need to, or at least in the current implementation. Can you suggest a use case for this ?